### PR TITLE
feat : Finished Wishlist and Cart Page

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
Finished Wishlist and Cart Page.

### CODE REVIEW

File: Pages/Cart-Page/cart.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    <script defer src="../topnavbar.js"></script>
+    <title>Enztron Store | Cart</title>
+</head>
+<body>
+    <div class="top-bar">
+        <div class="left-topbar-container">
+            <button class="icon-btn"><i class="fa fa-bars" aria-hidden="true"></i></button>
+            <h2 id="top-bar-brand-name">Enztron Book Store</h2>
+            <div class="search-bar">
+                <input class="search-bar-input" placeholder="Search"/>
```
Comment (rahulk31) [inline Pages/Cart-Page/cart.html:26]: font-family: inherit;
For the input, font-family can be inherited to make the font-style consistent throughout the page.

File: styles.css [hunk 2]
```diff
@@ -104,6 +109,11 @@ a:hover::after
...
 }
 
+#top-bar-brand-name
+{
+    cursor: pointer;
+}
+
 .search-bar .search-bar-input
 {
     outline: none;
```
Comment (rahulk31) [inline styles.css]: wrt class: .search-bar-input
Suggestion: font-family: inherit;
For the input, font-family can be inherited to make the font-style consistent throughout the page.

**Review by rahulk31 — COMMENTED** [review]
Could not find any other flaw 🧐, Great Work. UI is so neat.

**Comment by nikhiljugale007** [comment]
Code and UI looks good. 
Good to merge.

File: styles.css (hunk 2 above)
Comment (Naman-Saxena1) [reply styles.css]: Ooh right, I totally forgot about font-family of search bar, thanks!

**Comment by Naman-Saxena1** [comment]
> Could not find any other flaw 🧐, Great Work. UI is so neat.

Thanks! 😄

**Comment by Naman-Saxena1** [comment]
> The wish list page looks great syntactically and design wise. Recommendation: Can center the cards on the page to maybe look somewhat like this since there is a lot of area left on RHS: 

I didn't noticed that, thanks for pointing out.

**Pull request merged**